docs/tfengine/schemas/monitor.md

```diff
@@ -28,6 +28,12 @@ Domain for the Forseti instance.
 
 Type: string
 
+### forseti.network
+
+Name of the bastion host's network.
```

Comment:
Copy-paste error, should be "Forseti", not "bastion host". Same in `forseti.subnet`.